### ISSUE TITLE
Dhorme Khimaira/Apademak Crashing Client

### ISSUE THREAD

**Issue by iMorgoth** [LegionDark/Issues#1257]
**Date & Time**: 
 11/29/16 - All day

**FFXI Client Version (use `/ver`)**: 
Current - 30161004_1

**Server's Expected Client Version matches, yes/no? (use `$ecv`)**: 
Yes

**Character Name**: 
Varda

**Nation**: 
Bastok

**Job(level)/Sub Job(level)**: 
THF99/DNC49

**NPC or Monster or item Name**: 
Dhorme Khimaira/Apademak

**Zone name**: 
Abyssea-Ulegrand

**Coordinates (use `$where`)**: 
X-270   Y -156   Z-281  (121)

**Multi-boxing? (multiple clients on same connection)**: 
Yes, but tried with someone not Dual boxing and happened to them

**Steps To Reproduce / any other info**: 
Fighting Dhorme Khimaira for more than 3 minutes. The screen freezes and I need to completely restart the client. Happened to several individuals, and cannot kill the NM due to the multiple disconnects. Seems to only be these two NMs as I can farm the pops and not DC at all

**Comment by Heepster03** [comment]
Appears to be from Sudden Lunge spam.

**Comment by TeoTwawki** [comment]
@Heepster03 no, it is not. https://github.com/LegionDark/Issues/search?q=stun&type=Issues&utf8=%E2%9C%93 its any stun that causes "sudden lunge issues"
